src/core/data.jl

```diff
@@ -455,7 +455,7 @@ end
 # what are the values for delta_re and ne
 ""
 function ss_top_oil_rise(branch, result; base_mva=100, delta_rated=75)
-    if !branch["transformer"]
+    if !(br["type"] == "transformer" || br["type"] == "xf")
```

Comment:
For sure this will not work becouse `br` is not defined.  It might work if `br` is changed to `branch`.  As a general comment I would recomend leveraging the PowerModels data standards when possible.  In this case `branch["transformer"] = true/false` would be preferable to `branch["type"] = "transformer"/"xf"`.

Reply:
Is there a preferred way to indicate if a branch is a series capacitor? Not common, though it does appear on at least one GIC test case.

Reply:
Nope, free for you to do.  I do wonder if a series capacitor is just a special case of the branch model.

Reply:
A series cap should just be a line with negative X in the positive sequence representation. For the GIC model, it will be an open circuit instead of a resistance + series voltage though